dashboard/src/pages/ingest/sips/index.vue

```diff
@@ -19,12 +19,14 @@ import { useLayoutStore } from "@/stores/layout";
 import { useSipStore } from "@/stores/sip";
 import IconInfo from "~icons/akar-icons/info-fill";
 import IconAll from "~icons/clarity/blocks-group-line?raw&font-size=20px";
-import IconQueued from "~icons/clarity/clock-line?raw&font-size=20px";
 import IconClose from "~icons/clarity/close-line";
-import IconError from "~icons/clarity/remove-line?raw&font-size=20px";
+import IconError from "~icons/clarity/flame-line?raw&font-size=20px";
+import IconQueued from "~icons/clarity/hourglass-line?raw&font-size=20px";
+import IconFailed from "~icons/clarity/remove-line?raw&font-size=20px";
 import IconSearch from "~icons/clarity/search-line";
 import IconDone from "~icons/clarity/success-standard-line?raw&font-size=20px";
 import IconInProgress from "~icons/clarity/sync-line?raw&font-size=20px";
+import IconWarning from "~icons/clarity/warning-standard-line?raw&font-size=20px";
```

Comment:
I was trying to name the icons based on their purpose, I'd use `IconPending` here and rename the others too.